src/components/andresin87/utils/icons/index.js

```diff
@@ -0,0 +1,21 @@
+import iconHome from './iconHome.jsx'
+import iconHomeOutline from './iconHomeOutline.jsx'
+import iconGenerate from './iconGenerate.jsx'
+import iconCopy from './iconCopy.jsx'
+import iconSun from './iconSun.jsx'
+import iconMoon from './iconMoon.jsx'
+import iconUsers from './iconUsers.jsx'
```

Comment:
Hola!! 👋 El error está en esta linea cuando se quiere hacer el `build` del proyecto, el fichero `IconUsers` empieza en mayúscula y su import está en minúscula, por lo que no encuentra ese fichero  😊

```suggestion
import iconUsers from './IconUsers.jsx'
```

Reply:
thks

Reply:
done